tests/tbasic

```diff
@@ -147,13 +147,13 @@ title PARA "Test fetching public keys without PIN"
 ORIG_OPENSSL_CONF=${OPENSSL_CONF}
 sed "s/^pkcs11-module-token-pin.*$/##nopin/" ${OPENSSL_CONF} > ${OPENSSL_CONF}.nopin
 OPENSSL_CONF=${OPENSSL_CONF}.nopin
-echo "" | ossl 'pkey -in $PUBURI -pubin -passin stdin -pubout -out ${TMPPDIR}/rsa.pub.nopin.pem'
-echo "" | ossl 'pkey -in $ECPUBURI -pubin -passin stdin -pubout -out ${TMPPDIR}/ec.pub.nopin.pem'
+echo "" | ossl 'pkey -in $PUBURI -pubin -pubout -out ${TMPPDIR}/rsa.pub.nopin.pem'
+echo "" | ossl 'pkey -in $ECPUBURI -pubin -pubout -out ${TMPPDIR}/ec.pub.nopin.pem'
```

Comment:
do we still need the `echo ""` here when no prompt should be shown?

Reply:
Not needed indeed, remove and added separate PARA for prompting test